README.md

```diff
@@ -12,7 +12,7 @@ Setup
 
 Using pip:
 ```
-pip install ansible-lint
+pip2.7 install ansible-lint
```

Comment:
Can we just make this `pip2` rather than `pip2.7`. I think that should have the same result.

Just in case anyone is running it on RHEL6.